app/src/main/java/org/mozilla/focus/utils/AppConfigWrapper.java

```diff
@@ -33,6 +35,11 @@
     private static final boolean SURVEY_NOTIFICATION_ENABLED = false;
     static final int DRIVE_DEFAULT_BROWSER_FROM_MENU_SETTING_THRESHOLD = 2;
 
+    private static final Map<String, String> lifeFeedProviderConfigNameMapping = new HashMap<>();
+    static {
```

Comment:
please add some comment to it. I also feel it's like you are doing a data migration, you'll probably need a uni test for this to avoid future breakage.

Reply:
Or at least create an issue and make the unit test the priority for next sprint.

Reply:
I had an impression we previously add the priority of how we can override news source. Maybe it's worth looking when you want to add a default value.

Reply:
I've filed the issue to add unit test #3670